### ISSUE TITLE
concat error

### ISSUE THREAD

**Issue by shibiaoz** [jonkemp/gulp-useref#205]
concat two files ,but there is no semicolon between different files，
code in html 
```
<!-- build:js scripts/combined.js -->
    <script type="text/javascript" src="scripts/one.js"></script> 
    <script type="text/javascript" src="scripts/two.js"></script> 
    <!-- endbuild -->
```
code in one.js 
```
;(function (){
	console.log('onew')
})(window)
```
code in two.js 
```
(function() {
	console.log('two')
})(window)
```

combined.js
```
;(function (){
	console.log('onew')
})(window)
(function() {
	console.log('two')
})(window)
```
so how to add semicolon between files ,if the last files do not have semicolon in header

**Comment by shibiaoz** [comment]
the combined.js is error in browser

**Comment by jonkemp** [comment]
There is an option for this now. It's not been published to npm yet. Soon hopefully.

**Comment by jonkemp** [comment]
Published v3.1.0.

**Comment by shibiaoz** [comment]
ok,i try it,now it is right ,thank you @jonkemp
